src/classes/test_multiple_inheritance.py

```diff
@@ -9,13 +9,18 @@
 
 def test_multiple_inheritance():
     """Multiple Inheritance"""
-
+    class Parent:
```

Comment:
In this example you'll have to test multiple inheritance, while you are testing only simple inheritance.